bundle/manifests/clusterlogging.clusterserviceversion.yaml

```diff
@@ -135,6 +135,7 @@ metadata:
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-logging
     operators.openshift.io/infrastructure-features: '["disconnected","proxy-aware"]'
+    operators.openshift.io/must-gather-image: registry.redhat.io/openshift-logging/cluster-logging-rhel9-operator@sha256:b3a09d6dfc5109cd5d3ebf24201f9d0549d77f8965907c1b6b15faba15c767eb
```

Comment:
This is an upstream repo and this should point to the upstream image.  Given this annotation is meaningless (I think) to the upstream version of the bundle, I would propose simply fixing it in productization scripts and forgo making any change here

Reply:
Will look at incorporating this into midstream.